c2corg_ui/static/js/alertservice.js

```diff
@@ -49,11 +49,8 @@ app.Alerts.prototype.add = function(data) {
   var msg = data['msg'];
   msg = msg instanceof Object ? this.formatErrorMsg_(msg) :
       this.filterStr_(msg);
-  this.alerts_.push({
-    type: data['type'] || 'warning',
-    msg: msg,
-    timeout: timeout
-  });
+  data['type'] = data['type'] || 'warning';
+  this.alerts_.push(data);
```

Comment:
This change is not correct: ``timeout`` and ``msg`` changed a few lines above are no longer propagated. Why do you need to do that change?